src/dropdown/dropdown.ts

```diff
@@ -7,14 +7,20 @@ import {Directive, Input, Output, HostListener, EventEmitter} from '@angular/cor
   selector: '[ngbDropdown]',
   exportAs: 'ngbDropdown',
   host: {
-    'class': 'dropdown',
+    '[class.dropdown]': '!up',
+    '[class.dropup]': 'up',
     '[class.open]': 'isOpen()',
     '(keyup.esc)': 'closeFromOutside()',
     '(document:click)': 'closeFromOutside()'
   }
 })
 export class NgbDropdown {
   /**
+   * Indicates that dropdown should in fact drop up
```

Comment:
I would say "Indicates that the dropdown should open upwards" or something similar.